.rubocop.yml

```
@@ -404,13 +404,13 @@ Layout/SpaceAroundEqualsInParameterDefault:
     - no_space
 
 Layout/SpaceBeforeBlockBraces:
-  EnforcedStyle: no_space
+  EnforcedStyle: space
   SupportedStyles:
     - space
     - no_space
 
 Layout/SpaceInsideBlockBraces:
-  EnforcedStyle: no_space
+  EnforcedStyle: space
```

```suggestion
  EnforcedStyle: no_space
```
Bên trong block thì k cần space đầu cuối